### ISSUE TITLE
[TTPReq] Education: Accept contactless payments with only an iPhone

### ISSUE THREAD

**Issue by staskus** [woocommerce/woocommerce-ios#14161]
Designs & Copy: pdfdoF-5KF-p2
Flows: pdfdoF-5GF-p2
Depends on: https://github.com/woocommerce/woocommerce-ios/issues/14159

## Education Step Screen Functionality

If it's not already done, I suggest creating a reusable full-screen scroll view to fit as much educational content as needed. Support text blocks, or numbered text blocks as we have in 'Set up Tap to Pay'.

## Accept contactless payments with only an iPhone

- Use the visual assets from [Marketing Guidelines](https://developer.apple.com/tap-to-pay/marketing-guidelines/)
- Use the latest bezels
- Use localized images for all the supported TTP markets

## Designs

![Image](https://github.com/user-attachments/assets/edaf35f3-4bb6-4bb3-88f8-9ba4032dd8c6)
